solutions/59.js

```diff
@@ -0,0 +1,15 @@
+//Function that takes in a number, returns true if number is prime. False otherwise.
```

Comment:
add comments to describe your function like this guy: https://github.com/llipio/algorithms/pull/55/files